db.go

```diff
@@ -161,6 +168,14 @@ func newDBMetrics(db *DB, r prometheus.Registerer) *dbMetrics {
 		Name: "prometheus_tsdb_tombstone_cleanup_seconds",
 		Help: "The time taken to recompact blocks to remove tombstones.",
 	})
+	m.storageBytes = prometheus.NewGauge(prometheus.GaugeOpts{
+		Name: "prometheus_tsdb_storage_blocks_bytes_total",
+		Help: "The number of bytes that are currently used for local storage by blocks.",
+	})
+	m.dataLimitDeletions = prometheus.NewCounter(prometheus.CounterOpts{
+		Name: "prometheus_tsdb_size_limit_deletions_total",
+		Help: "The number of times that blocks were deleted because the maximum number of bytes was exceeded.",
```

Comment:
Just curious, where do you think this metric could be useful?

Reply:
I think less so the number of deletions that occurred, but more so that deletions are occurring. It would let the user know that the cap is being reached, and that data is removed because of the cap, not because of the retention period being hit.